Bugzilla/Install/Localconfig.pm

```diff
@@ -40,10 +40,7 @@ our @EXPORT_OK = qw(
 
 sub _sensible_group {
     return '' if ON_WINDOWS;
-    my @groups     = qw( apache www-data _www );
```

Comment:
This is a permanent change to how localconfig works. Is it called out sufficiently in docs or whatever, that it will stop defaulting to group apache *over* EGID?